src/NzbDrone.Core/Notifications/Discord/DiscordSettings.cs

```diff
@@ -19,7 +19,7 @@ public class DiscordSettings : IProviderConfig
         public DiscordSettings()
         {
             // Set Default Fields
-            GrabFields = new[] { 0, 1, 2, 3, 5, 6, 7, 8, 9 };
```

Comment:
I am not a C# expert, but it seems odd to store the default field types as an int[] and constantly cast to the `DiscordGrabFieldType` enum. Is this done for some reason I don't understand with the frontend?

Reply:
I believe this is mostly a limitation of our dynamic handling of settings when serialized for the API, not really a burden to cast as necessary since it makes the conversion via the API painless.